hack/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+rm snap-plugin-collector-rando
```

Comment:
Is a makefile coming up? Otherwise, it would probably make sense to take a look at `cmake`